model/DataQuery.php

```diff
@@ -779,6 +779,22 @@ public function selectFromTable($table, $fields) {
 	}
 
 	/**
+	 * Add the given fields from the given table to the select statement.
+	 *
+	 * @param String $table Unquoted table name (will be escaped automatically)
+	 * @param Array $fields Database column names (will be escaped automatically)
+	 */
+	public function addSelectFromTable($table, $fields) {
```

Comment:
New public method probably means it needs to be a new minor (feature) version. Can we PR to 3 branche instead of 3.5?